hashdist/cli/frontend_cli.py

```diff
@@ -123,7 +123,16 @@ class Develop(ProfileFrontendBase):
     """
     Builds a development profile in the HashDist YAML profile spec
     format, at the same location as the profile yaml file, but without
-    the .yaml suffix.
+    the .yaml suffix. You can use the 'target' argument to name it differently
+    than your profile yaml file.
+
+    Example:
+
+        hit develop default.yaml xx
+
+    This creates a throw away developmet profile 'xx', into which you can
```

Comment:
I wouldn't phrase it like this.  The point of a development profile is that it is NOT built in the HashDist cache, it's built in the target directory.  As a result:

* You can install other things into this profile (pip install works).  None of these things are tracked/saved by HashDist
* The profile is a directory, not a symbolic link back to the cache.